Dockerfile-index

```diff
@@ -0,0 +1,50 @@
+# Process for indexing papers by arXiv ID.
+#
+# Expects a path to a list of newline-delimited arXiv IDs (versionless).
+#
+# Example (local stack):
+#
+#    $ mkdir /tmp/to_index
+#    $ cp arxiv_id_dump.txt /tmp/to_index
+#    $ docker run -it --network=arxivsearch_es_stack \
+#    >   -v /tmp/to_index:/to_index \
+#    >   -e ELASTICSEARCH_HOST=elasticsearch \
+#    >   arxiv/search-index /to_index/arxiv_id_dump.txt
+#
+# See also ELASTICSEARCH_* and METADATA_ENDPOINT parameters, below.
+
+FROM arxiv/base
+
+# Add Python consumer and configuration.
+ADD requirements/prod.txt /opt/arxiv/requirements.txt
+ADD app.py /opt/arxiv/
+RUN pip install -U pip
+RUN pip install -r /opt/arxiv/requirements.txt
+
+ENV PATH "/opt/arxiv:${PATH}"
+
+ADD schema /opt/arxiv/schema
+ADD mappings /opt/arxiv/mappings
+ADD search /opt/arxiv/search
+ADD tests /opt/arxiv/tests
+ADD bulk_index.py /opt/arxiv/
+
+WORKDIR /opt/arxiv/
+
+ENV LC_ALL en_US.utf8
+ENV LANG en_US.utf8
+ENV LOGLEVEL 10
```

Comment:
Do we have standard log rotation or an override on production log level setting to maintain disk space, or just plenty of disk space relative to the output volume at expected loads?

Reply:
The on premises logs are rotated daily and overwritten after two weeks. A few of those logs (like the main, non-error web access logs) are permanently archived.

Reply:
https://github.com/cul-it/arxiv-search/issues/110